.github/workflows/android_build.yaml

```diff
@@ -16,4 +17,28 @@ jobs:
           java-version: 11
 
       - name: Run tests
-        run: ./gradlew test
\ No newline at end of file
+        run: ./gradlew test
+
+  instrumentation_tests:
+    name: Instrumentation Tests
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        api-level: [26]
```

Comment:
Cool stuff with the matrix! I had not thought of playing around with it. Why not put in our lowest supported API and our latest? That would be 31.

Reply:
I left it as version 26 since I had some issue with version 30. I can try adding in version 31 and see if it works 👍